renovate.json

```diff
@@ -164,7 +164,6 @@
         "@jest/console",
         "@jest/reporters",
         "@jest/types",
-        "@types/jest",
```

Comment:
breaking change in a patch version..I removed this for now to see if it gets fixed upstream.